django/contrib/admin/static/admin/css/base.css

```diff
@@ -24,7 +24,7 @@ html[data-theme="light"],
     --breadcrumbs-link-fg: var(--body-bg);
     --breadcrumbs-bg: #264b5d;
 
-    --link-fg: #417893;
```

Comment:
This is not a proper solution. `#417893` is too bright and was already rejected in the previous PR, see [comment](https://code.djangoproject.com/ticket/34038#comment:5).

Reply:
Did you mean #437c98 is too bright? If so, then I guess this means the problem was already solved in #16378 and the ticket should be closed then unless I'm missing something.

Reply:
Yes, I closed the ticket and asked for a clarification from the accessibility team.

Thanks for this patch :+1: